test/AzureExtension/AzureExtension.Test.csproj

```diff
@@ -5,7 +5,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <RootNamespace>AzureExtension.Test</RootNamespace>
-    <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+    <RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>

```

Comment:
Why switch these ones to `win` from `win10` and then in `DevHome.Logging` switch from `win10` to `win`?

Reply:
Should be win10, as per the [DevHome RidGraph fix ](https://github.com/microsoft/devhome/blob/eb5273b1166e12eab6a825af391674458b026210/test/DevHome.Test.csproj). Has been changed to win10 - thanks!